components/TypeForm/PopUp.tsx

```diff
@@ -20,7 +19,7 @@ const PopUp = ({
   return (
     <PopupButton
       onClose={handleClose}
-      buttonProps={{ type: 'button', className: className }}
+      buttonProps={{ type: 'button', id, className }}
```

Comment:
The id has to be passed in this way now. Otherwise it doesn't get through to the button element and the focus management didn't work. It must have changed in some previously version of React Typeform embed and broke this 🤔

Reply:
To test this, click on the "Take my exam" button
https://deploy-preview-307--giving-campaign.netlify.app/stories/education-for-a-life-of-purpose

Click escape and see that the focus is on the button after the modal closes.